packages/atomic/src/utils/initialization-utils.tsx

```diff
@@ -82,10 +82,25 @@ export interface InitializableComponent extends ComponentInterface {
   error: Error;
 }
 
+/**
+ * Makes Shadow Dom elements compatible with the focus-visible polyfill https://github.com/WICG/focus-visible
+ * Necessary for Safari under version 15.4.
+ */
 export function applyFocusVisiblePolyfill(element: HTMLElement) {
-  if (window.applyFocusVisiblePolyfill && element.shadowRoot) {
+  if (!element.shadowRoot) {
+    return;
+  }
+
+  if (window.applyFocusVisiblePolyfill) {
     window.applyFocusVisiblePolyfill(element.shadowRoot);
+    return;
   }
+
+  window.addEventListener(
```

Comment:
when lazy loaded this is necessary https://github.com/WICG/focus-visible#lazy-loading